framework/src/application/schema/constants_schema.js

```diff
@@ -20,15 +20,7 @@ module.exports = {
 	required: [
 		'activeDelegates',
 		'standbyDelegates',
-		'blockSlotWindow',
-		'blockReceiptTimeout',
-		'maxPayloadLength',
-		'maxSharedTransactions',
-		'maxVotesPerAccount',
 		'totalAmount',
-		'transactionTypes',
-		'unconfirmedTransactionTimeout',
-		'expiryInterval',
 		'delegateListRoundOffset',
```

Comment:
I don't understand, why you removed a few constants and kept a few?

Reply:
it was removed because it was not used anywhere